app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -5,24 +5,32 @@ class Proofer
     AUTH_TOKEN_CACHE_KEY = :usps_ippaas_api_auth_token
     # Automatically refresh our auth token if it is within this many minutes of expiring
     AUTH_TOKEN_PREEMPTIVE_EXPIRY_MINUTES = 1.minute.freeze
+    # Assurance Level to use for enhanced in-person proofing
+    USPS_EIPP_ASSURANCE_LEVEL = '2.0'
 
     # Makes HTTP request to get nearby in-person proofing facilities
-    # Requires address, city, state and zip code.
+    # Requires address, city, state, zip code, and is_enhanced_ipp.
     # The PostOffice objects have a subset of the fields
     # returned by the API.
     # @param location [Object]
+    # @param is_enhanced_ipp [Boolean]
     # @return [Array<PostOffice>] Facility locations
-    def request_facilities(location)
+    def request_facilities(location, is_enhanced_ipp)
       url = "#{root_url}/ivs-ippaas-api/IPPRest/resources/rest/getIppFacilityList"
-      body = {
+      request_body = {
         sponsorID: sponsor_id,
         streetAddress: location.address,
         city: location.city,
         state: location.state,
         zipCode: location.zip_code,
-      }.to_json
+      }
+
+      if is_enhanced_ipp
+        request_body[:IPPAssuranceLevel] = USPS_EIPP_ASSURANCE_LEVEL
+        request_body[:sponsorID] = IdentityConfig.store.usps_eipp_sponsor_id.to_i
+      end
```

Comment:
I don't think you need to add `IPPAssuranceLevel` to this request body.  I think this is only for enrollments.